docker/pulp/Dockerfile

```diff
@@ -6,38 +6,49 @@ ENV PYTHONUNBUFFERED=1 \
     PULP_VENV=/venv \
     PULP_SETTINGS=/etc/pulp/settings.py \
     DJANGO_SETTINGS_MODULE=pulpcore.app.settings \
-    ANSIBLE_TEST_VENV=/venv_ansible_test
+    ANSIBLE_TEST_VENV=/venv_ansible_test \
+    ENABLED_COLLECTIONS="rh-python36 rh-postgresql12"
 
-RUN yum -y install epel-release \
+
+RUN yum -y install centos-release-scl-rh \
     && yum -y install \
+        gcc \
```

Comment:
Is it the PyYaml install that requires gcc?

Reply:
It's psycopg2 package. `pulp-ansible` switched their dependency from `psycopg2-binary` to `psycopg2` [1], which is an sdist package that requires native extensions to be built during the installation.

[1]https://github.com/pulp/pulpcore/commit/3dfd06745da718f529cf8a31b16281a5b3c04de3